staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -63,12 +65,19 @@ import (
 )
 
 var (
-	fakeSchema                = sptest.Fake{Path: filepath.Join("..", "..", "..", "testdata", "openapi", "swagger.json")}
-	testingOpenAPISchemas     = []testOpenAPISchema{AlwaysErrorsOpenAPISchema, FakeOpenAPISchema}
+	fakeSchema          = sptest.Fake{Path: filepath.Join("..", "..", "..", "testdata", "openapi", "swagger.json")}
+	fakeOpenAPIV3Legacy = sptest.OpenAPIV3Getter{Path: filepath.Join("..", "..", "..", "testdata", "openapi", "v3", "api", "v1.json")}
+	fakeOpenAPIV3AppsV1 = sptest.OpenAPIV3Getter{Path: filepath.Join("..", "..", "..", "testdata", "openapi", "v3", "apis", "apps", "v1.json")}
+	// testingOpenAPISchemas = []testOpenAPISchema{FakeOpenAPISchema}
```

Comment:
Should we remove this commented out line?

Reply:
Fixed.